cdap-cli/src/main/java/co/cask/cdap/cli/english/Article.java

```diff
@@ -39,4 +40,13 @@ public String toString(Word word) {
     }
   }
 
+  /**
+   * "the"
+   */
+  private static class The extends Article {
```

Comment:
what's the point of this? It seems that it can just be the string constant "the". Maybe I am missing something.